terminus-ui/src/select/select.component.ts

```diff
@@ -204,6 +206,26 @@ export class TsSelectComponent extends TsReactiveFormBaseComponent {
   public selectionChange: EventEmitter<any[]> = new EventEmitter();
 
 
+  constructor(
+    private changeDetectorRef: ChangeDetectorRef,
+  ) {
+    super();
+  }
+
+
+  /**
+   * Trigger change detection when the underlying form changes
+   */
+  public ngOnInit(): void {
+    // istanbul ignore else
+    if (this.formControl) {
+      this.formControl.valueChanges.subscribe((v) => {
+        this.changeDetectorRef.detectChanges();
```

Comment:
unsubscribe?

Reply:
Dammit. Knew I was going to forget something. Thanks for the catch.. fix incoming